app/assets/stylesheets/questions.css.scss

```diff
@@ -84,6 +84,10 @@
   }
 }
 
+#top-padding {
```

Comment:
As mentioned [in another PR recently](https://github.com/penzance/askup/pull/82#discussion_r59989812) (please refer to that for an extended explanation), we usually try to use semantic class names for styling, instead of style-indicating class names. Also using an ID called `top-padding` is a bit confusing -- an id would usually give you semantic information about what that element and its function are. Classes, while still semantic, are probably a better approach -- or naming the ID for the semantic purpose of the element instead of its preferred styling.